Card/src/main/java/com/paypal/android/card/CardAPIRequestFactory.kt

```diff
@@ -35,4 +35,43 @@ internal class CardAPIRequestFactory {
         val path = "v2/checkout/orders/$orderID/confirm-payment-source"
         return APIRequest(path, HttpMethod.POST, body)
     }
+
+    fun createAuthorizeWith3DSVerificationRequest(orderID: String, card: Card): APIRequest {
+        val cardNumber = card.number.replace("\\s".toRegex(), "")
```

Comment:
Should this be changed to remove any non-number? And should this logic for cleaning up the card number be moved to a function that can be used anywhere it needs to be run? (I assume non-3ds card flows will also need to do it)

Reply:
We may be missing a spec for this. For now all it does is strip out spaces, but we should have local validation in place to provide instant feedback.